File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/session/model/Session.kt
package pt.isel.leic.ptgest.domain.session.model

import pt.isel.leic.ptgest.domain.session.SessionType
import java.util.*

data class Session(
    val id: Int,
    val beginDate: Date,
    val endDate: Date?,
    val type: SessionType,
    val cancelled: Boolean = false
)

File: code/js/src/views/user/trainerViews/models/sessions/Session.ts
class Session {
    id: number
    beginDate: string | null
    endDate: string | null
    type: string
    cancelled: boolean = false

    constructor(id: number, beginDate: string | null, endDate: string | null, type: string, cancelled: boolean) {
        this.id = id
        this.beginDate = beginDate
        this.endDate = endDate
        this.type = type
        this.cancelled = cancelled
    }
}
export default Session
